CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+##2016-07-XX (TBD) - Release 2.2.0
+### Summary
+This release includes new features as well as bug fixes.
+
+The overview data model now contains more information. It used to only contain the top 10 nodes based on an issue count.
+Now the overview data model contains information about all nodes the overview is based on and with more details per node
```

Comment:
add some commas:

`Now, the overview data model contains information about all nodes the overview is based on, and`